package.json

```diff
@@ -19,6 +19,7 @@
     "build:cjs": "ttsc -p tsconfig.prod.json --module commonjs --outDir lib/cjs && tsc-alias -p tsconfig.prod.json --outDir lib/cjs",
     "build": "npm-run-all build:*",
     "test": "test-storybook",
+    "test-jest": "jest",
```

Comment:
Does this mean we keep all SB tests and duplicate them as Jest tests?

Reply:
No, some of these (interaction) tests really belong in storybook where as other (unit, integration) are better suited for jest. The line is blurry, but the advantage of having some tests in storybook  is that the intended interactions are very clear *and* they're visually obvious what is going wrong. With jest tests, debugging tests tends to be quite painful since you don't know what's actually on the browser screen.

Reply:
Ok makes sense